redfish-core/lib/sensors.hpp

```diff
@@ -551,8 +551,6 @@ void getChassis(const std::shared_ptr<SensorsAsyncResp>& sensorsAsyncResp,
         }
         if (chassisPath == nullptr)
         {
-            messages::resourceNotFound(sensorsAsyncResp->asyncResp->res,
```

Comment:
Since you removed the resource not found error message, what happens if people specify an invalid chassis?

I assume the Redfish request should not silently fail/return no output.

e.g. `https://${bmc}/redfish/v1/Chassis/this_chassis_does_not_exist/Sensors`